fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/PushDownLimitJoinRule.java

```diff
@@ -44,6 +44,14 @@ public boolean check(OptExpression input, OptimizerContext context) {
             return Lists.newArrayList(child);
         }
 
+        if (joinType.isSemiAntiJoin()) {
```

Comment:
This judgment does not seem to go, because there is an if (joinType.isInnerJoin() || joinType.isSemiAntiJoin()) before, and return ends the function